src/analyze/analyze-condition.c

```diff
@@ -72,18 +73,35 @@ static int log_helper(void *userdata, int level, int error, const char *file, in
         return r;
 }
 
-int verify_conditions(char **lines, UnitFileScope scope) {
+int verify_conditions(char **lines, UnitFileScope scope, const char *root) {
         _cleanup_(manager_freep) Manager *m = NULL;
+        _cleanup_free_ char *var = NULL;
+        _cleanup_free_ char **filenames = NULL;
```

Comment:
`_cleanup_strv_free_`?

Reply:
We don't actually `strdup` stuff into the strv so I think we're good with just `_cleanup_free_` here.